src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,29 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = factory.openSession();
+        Transaction transaction = session.beginTransaction();
+        try {
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            transaction.rollback();
```

Comment:
When you move transaction begin logic to try block it can be null here if something happened during starting a transaction so check for null here.